includes/fields/class-gravityview-field-approval.php

```
@@ -88,32 +88,64 @@ private function add_hooks() {
-	 * @param array                         $form Gravity Forms form array
-	 * @param int                           $entry_id Gravity Forms Entry ID
-	 * @param GravityView_Edit_Entry_Render $Edit_Entry_Render
+	 * @param array                         $form              Gravity Forms form array.
+	 * @param int                           $entry_id          Gravity Forms Entry ID.
+	 * @param GravityView_Edit_Entry_Render $edit_entry_render The Edit Entry renderer.
+	 * @param GravityView_View_Data         $gv_data           The GravityView View data.
 	 *
 	 * @return void
 	 */
-	public function update_edit_entry( $form = array(), $entry_id = 0, $Edit_Entry_Render = null ) {
+	public function update_edit_entry( $form = [], $entry_id = 0, $edit_entry_render = null, $gv_data = null ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$edit_entry_render'. (Unused Code Rules)

(UnusedFormalParameter)